### PR TITLE
Sound feedback

### PR DESCRIPTION
**Pull request by dderancourt**
US 41, sound feedback when player can't make a move

### CODE REVIEW

**Review by Gumatan — DISMISSED** [review]

**Review by LoicBrassart — DISMISSED** [review]
SnE

**Review by TheoBoschetti — DISMISSED** [review]
ok

File: frontend/src/Components/Game.jsx [hunk 1]
```diff
@@ -11,6 +11,15 @@ class Game extends React.Component {
       x: 3,
       y: 4,
       keyName: "ArrowDown",
+      canMove: true,
```
Comment (LoicBrassart) [inline frontend/src/Components/Game.jsx:18]: Your file is broken (just checked after a `git pull` on my machine)

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Evil spotted!

**Pull request merged**